server/start.go

```diff
@@ -208,31 +195,30 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 	return cmd
 }
 
-func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
-	addr := ctx.Viper.GetString(flagAddress)
-	transport := ctx.Viper.GetString(flagTransport)
-	home := ctx.Viper.GetString(flags.FlagHome)
+func startStandAlone(svrCtx *Context, appCreator types.AppCreator) error {
+	addr := svrCtx.Viper.GetString(flagAddress)
+	transport := svrCtx.Viper.GetString(flagTransport)
+	home := svrCtx.Viper.GetString(flags.FlagHome)
 
-	db, err := openDB(home, GetAppDBBackend(ctx.Viper))
+	db, err := openDB(home, GetAppDBBackend(svrCtx.Viper))
```

Comment:
@alexanderbez unrelated but a way to inject your own openDB method / run a remote db would be absolutely fire. 

I can make an issue / pr if you think it's a feature that'd you'd be open to.

Reply:
Sure! But I think in the very near future we're going to stick with a single DB -- RocksDB. Now I suppose we can still make it pluggable via an interface.

Reply:
even if with a single db, we still need a custom opener to customize db options.